Tests/NIOPosixTests/SystemCallWrapperHelpers.swift

```diff
@@ -17,16 +17,16 @@ import Foundation
 @testable import NIOPosix
 #endif
 
-public func measureRunTime(_ body: () throws -> Int) rethrows -> TimeInterval {
-    func measureOne(_ body: () throws -> Int) rethrows -> TimeInterval {
-        let start = Date()
+public func measureRunTime(_ body: () throws -> Int) rethrows -> UInt64 {
```

Comment:
It will probably make your life easier to keep the function signature the same, and just compute the `TimeInterval` internal to `measureOne`.